dev-libs/weston/weston-1.12.0.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
could you please look into bumping to EAPI 6 here? see also https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/